src/decorators/pure.js

```diff
@@ -1,8 +1,17 @@
 import t from 'tcomb';
 import isReactComponent from '../isReactComponent';
 import debug from 'debug';
+import isEqual from 'lodash/isEqual';
 
+const testLogsMap = {};
 const log = debug('revenge:@pure');
+const logTest = (rootNodeID, displayName, key) => {
+  const component = `${displayName}-${rootNodeID}`;
+  if (!testLogsMap[component]) {
+    testLogsMap[component] = true;
+    console.warn(`Update avoidable! (component: ${displayName}, key: ${key}, rootNodeID: ${rootNodeID})`);
```

Comment:
this logs a single warning per component, but there could be more than one, given more re-renders right?